securityscanutils/github_writer.go

```diff
@@ -83,6 +83,10 @@ func (g *GithubIssueWriter) CreateUpdateVulnerabilityIssue(ctx context.Context,
 		return nil
 	}
 
+	if developerDebugInstructions != "" {
+		vulnerabilityMarkdown = fmt.Sprintf("%s\n%s", developerDebugInstructions, vulnerabilityMarkdown)
```

Comment:
Do we want "## Debug Instructions" to be added by go-utils rather than the calling repo? I could go either way.

Reply:
But if the calling repo adds it, maybe the naming should be more generic for any additional context they want added to the GitHub issue. Actually, yeah, that's probably a better way to go in go-utils. Can we make the flag/var naming and description more general?

Reply:
Yeah good suggestion, I'll update